data/campaigns/World_Conquest/lua/map/scenario_utils/plot.lua

```diff
@@ -45,29 +45,29 @@ function add_plot(scenario, scenario_num, nplayers)
 	elseif scenario_num == 2 then
 		local r = wesnoth.random(nplayers)
 		start_message(r, true, _ "Ahhh. Just look at these fertile lands, ripe for conquest!")
-		start_message("4", true, _ "What's this?! Foreign invaders have set foot upon our shores!")
-		start_message("5", true, _ "We'll send them back to where they came from quick enough. All troops, to me!")
+		start_message("4", true, _ "What’s this?! Foreign invaders have set foot upon our shores!")
+		start_message("5", true, _ "We’ll send them back to where they came from quick enough. All troops, to me!")
 
 		end_message("1,2,3", false, _ "We win! Truly ours will be a glorious empire!")
 		end_message("1,2,3", true, _ "Do not get hasty! To reach the great continents of the east we must subdue the savage coast of Moragdu, notorious for its corsairs, before we can advance further.")
 	elseif scenario_num == 3 then
-		start_message("4", true, _ "What's this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
+		start_message("4", true, _ "What’s this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
 		start_message("5", true, _ "The time of prophecies is upon us! The destroyers have arrived from beyond the sea!")
-		start_message("6", true, _ "I don't know what the lizard is babbling about, but we'll sort these 'destroyers' out quickly enough.")
+		start_message("6", true, _ "I don’t know what the lizard is babbling about, but we’ll sort these ‘destroyers’ out quickly enough.")
 
 		end_message("1,2,3", true, _ "Victory is ours! Let us take sail in search of new lands to conquer!")
 	elseif scenario_num == 4 then
 		local r = wesnoth.random(nplayers)
 		start_message("5", true, _ "Ready yourselves, men! The conquering hordes are upon us!")
 		start_message("4", true, _ "Word has come to our island of your victories. Your army is impressive, but it will avail you not.")
 		start_message("6", true, _ "Indeed. You come this far, and no farther!")
-		start_message(r, true, _ "Hah! We didn't come all this way just to give up and go home. We will destroy you as we have destroyed all that have stood before us!")
+		start_message(r, true, _ "Hah! We didn’t come all this way just to give up and go home. We will destroy you as we have destroyed all that have stood before us!")
 		start_message("7", true, _ "So be it. We stand squarely behind our allies.")
-		start_message("2,3", true, _ "You are not the only ones with allies! We'll push on till the end!")
+		start_message("2,3", true, _ "You are not the only ones with allies! We’ll push on till the end!")
 
 		end_message("1,2,3", true, _ "Onwards to the ships!")
 		end_message("1,2,3", false, _ "We have travelled great oceans and brought low mighty empires. Should we not wait until regain our forces?")
```

Comment:
Should we not wait until *we* regain our forces?

I think adding the missing word helps, but it still doesn't sound right. Maybe "until our forces regain their strength".

Reply:
Thanks for highlighting this, I didn't read everything super-thoroughly. Agreed on your rewording.